internal/controllers/machine/machine_controller_phases_test.go

```diff
@@ -1100,3 +1100,182 @@ func TestReconcileInfrastructure(t *testing.T) {
 		})
 	}
 }
+
+func TestReconcileCertificateExpiry(t *testing.T) {
+	fakeTimeString := "2020-01-01T00:00:00Z"
+	fakeTime, _ := time.Parse(time.RFC3339, fakeTimeString)
+	fakeMetaTime := &metav1.Time{Time: fakeTime}
+
+	fakeTimeString2 := "2020-01-01T00:00:00Z"
+	fakeTime2, _ := time.Parse(time.RFC3339, fakeTimeString2)
+	fakeMetaTime2 := &metav1.Time{Time: fakeTime2}
```

Comment:
Why do we have those variables duplicate even though their content is identical? Should we change one of the timestamps?